jenkins/Jenkinsfile

```diff
@@ -385,24 +392,28 @@ pipeline {
                             script {
                                 String daisyImage = "gcr.io/compute-image-tools/daisy:latest"
                                 sh """
+                                    cd jenkins/gce-custom-images/cis-harden-images
+                                    git checkout $IMAGE_HARDENING_LIB_VERSION 
+
                                     # Create the Daisy scratch bucket if it doesn't exist. The Daisy workflow will clean it up at the end.
                                     gsutil ls $GCE_IMAGE_BUCKET || gsutil mb -p $GOOGLE_PROJECT -l $REGION $GCE_IMAGE_BUCKET
 
                                     docker pull $daisyImage
-                                    docker run -i --rm -v ${WORKSPACE}/jenkins/gce-custom-images:/daisy_source_files \
+                                    docker run -i --rm -v ${WORKSPACE}/jenkins/gce-custom-images:/gce-custom-images \
```

Comment:
if this gets much bigger it may be worth thinking of consolidating it along with the docker run command in `create_gce_image.sh`

Reply:
Yeah, that's a good idea.